### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by m-allanson**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Describe the issue that you're seeing.
 
 ### Steps to reproduce
 
-Clear steps describing how to reproduce the issue.
+Clear steps describing how to reproduce the issue. Please please please link to a demo project if possible, this makes your issue _much_ easier to diagnose (seriously).
```
Comment (jlengstorf) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I love this. 😄

**Comment by m-allanson** [comment]
Be clearer about requirements, remove unnecessary requirements and add some supporting info.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (m-allanson) [reply .github/ISSUE_TEMPLATE/bug_report.md]: 😄 hopefully gets the point across that a reproducible demo is the number one thing someone can do to help getting their issue triaged.

**Comment by jlengstorf** [comment]
Build passed, Gatsbygram failed.

**Pull request merged**